force-app/main/default/lwc/changeAmountOrFrequencyModal/changeAmountOrFrequencyModal.html

```diff
@@ -0,0 +1,25 @@
+<template>
+  <template if:true={open}>
+    <section role="dialog" tabindex="-1" aria-modal="true" aria-labelledby="modal-heading-01" class="slds-modal slds-fade-in-open">
+        <div class="slds-modal__container">
+          <button class="slds-button slds-button_icon slds-modal__close slds-button_icon-inverse" onclick={closeModal}>
+            <svg class="slds-button__icon slds-button__icon_large" aria-hidden="true">
+              <use xlink:href="/assets/icons/utility-sprite/svg/symbols.svg#close"></use>
+            </svg>
+            <span class="slds-assistive-text">Cancel and close</span>
+          </button>
+          <div class="slds-modal__header">
+            <h1 id="modal-heading-01" class="slds-modal__title slds-hyphenate">Change Amount or Frequency</h1>
```

Comment:
Strings should be Custom Labels so we can translate them (there are multiple here, and in other components, I'll just leave one comment)

Reply:
changed!